actionpack/lib/action_dispatch/journey/gtg/transition_table.rb

```diff
@@ -109,7 +109,7 @@ def visualizer(paths, title = "FSM")
           svg         = to_svg
           javascripts = [states, fsm_js]
 
-          # Annoying hack warnings
+          # Annoying hack warnings.
```

Comment:
Either remove this or explain what warnings are raised and why they're fixed by the code below.